advanced/is-pattern-1/templates/is/wso2is-pattern-1-identity-server-conf.yaml

```diff
@@ -42,32 +42,26 @@ data:
     [user_store]
     type = "database_unique_id"
 
-    [database.user]
```

Comment:
Why we have removed the user db

Reply:
In 6.0.0 also it was removed, since through 6.1.0 we haven't done changes to deployment toml, followed the same changes.
https://github.com/wso2/kubernetes-is/blob/6.0.x/advanced/is-pattern-1/templates/is/wso2is-pattern-1-identity-server-conf.yaml